packages/survey-creator-core/src/property-grid/index.ts

```diff
@@ -1477,7 +1469,7 @@ export abstract class PropertyGridEditor implements IPropertyGridEditor {
     surveyLocalization.currentLocale = prevCurrentLocale;
     this.onModalPropertyEditorShown(editor, property, question, options);
     options.onPropertyGridShowModalCallback(obj, property, question, surveyPropertyEditor, popupModel);
-    return surveyPropertyEditor;
+    return;
```

Comment:
This return statement does nothing, it might be here to make it explicit that we return nothing.